test/components/converters/test_pdfminer_to_document.py

```diff
@@ -185,3 +185,37 @@ def test_run_detect_paragraphs_to_be_used_in_split_passage(self, test_files_path
             "structure, allowing structure to emerge according to the \nneeds of the users.[1] \n\n"
         )
         assert docs["documents"][6].content == expected
+
+    def test_detect_undecoded_cid_characters(self):
```

Comment:
We should also add a test case to cover the warning in the run method with something like
```python
with caplog.at_level(logging.WARNING):
    output = converter.run(...)
    assert "..." in caplog.text
```
I would have expected it to fail because of `source=source,` is currently missing in the log message.